modules/serverless-knative-kafka-broker-metrics.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * serverless/observability/admin-metrics/serverless-admin-metrics.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="serverless-knative-kafka-broker-metrics_{context}"]
+= Knative Kafka broker metrics
+
+You can use the following metrics to debug and visualize the performance of Kafka broker.
+
+[cols=5*,options="header"]
+|===
+|Metric name
+|Description
+|Type
+|Tags
+|Unit
+
+|`event_count_1_total{job="kafka-broker-receiver-sm-service", namespace="knative-eventing"}`
+|Number of events received by a broker
+|Counter
+a|
+* `name` (broker name)
+* `namespace_name` (broker namespace)
+* `event_type` (event type)
+* `response_code` (HTTP response code returned by the broker)
+* `response_code_class` (HTTP response code class returned by the broker: 2xx, 3xx, 4xx, 5xx)
```

Comment:
Global comment: we should avoid using parens in this way. This looks like a description list, so I would suggest updating all of these lists as follows:

```suggestion
`name`:: broker name
`namespace_name`:: broker namespace
`event_type`:: event type
`response_code`:: HTTP response code returned by the broker
`response_code_class`:: HTTP response code class returned by the broker: 2xx, 3xx, 4xx, 5xx
```